packages/manager/modules/pci/src/projects/project/storages/containers/containers.html

```diff
@@ -57,40 +57,21 @@
                     class="text-truncate"
                 ></a>
             </oui-datagrid-column>
-            <oui-datagrid-column
-                data-title="'pci_projects_project_storages_containers_id_label' | translate"
-                data-property="id"
-                data-type="string"
-                data-sortable
-                data-filterable
-            >
-            </oui-datagrid-column>
             <oui-datagrid-column
                 data-title=":: 'pci_projects_project_storages_containers_region_label' | translate"
                 data-property="region"
                 data-type="string"
                 data-sortable
                 data-filterable
             ></oui-datagrid-column>
-            <oui-datagrid-column
-                data-title=":: 'pci_projects_project_storages_containers_containerType_label' | translate"
-                data-property="containerType"
-                data-sortable
-                data-type="string"
-            >
-                <span
-                    data-ng-if="$row.containerType"
-                    data-ng-bind="'pci_projects_project_storages_containers_containerType_' + $row.containerType | translate"
-                ></span>
-            </oui-datagrid-column>
             <oui-datagrid-column
                 data-title=":: 'pci_projects_project_storages_containers_offer_label' | translate"
                 data-type="string"
                 data-sortable
                 data-filterable
             >
                 <span
-                    data-ng-bind=":: 'pci_projects_project_storages_containers_offer_' + ($row.isHighPerfStorage ? 'high_performance' : 'swift') | translate"
+                    data-ng-bind=":: 'pci_projects_project_storages_containers_offer_' + ($row.s3StorageType == null ? 'swift' : ($row.isHighPerfStorage ? 'high_performance' :'standard_s3')) | translate"
```

Comment:
Better to create a const object with container types.

Reply:
Done Mohammed Thank you